src/api/app/controllers/webui/staging/projects_controller.rb

```diff
@@ -79,7 +78,7 @@ def preview_copy
       def copy
         authorize @staging_workflow
 
-        StagingProjectCopyJob.perform_later(@staging_workflow.project.name, params[:staging_project_project_name], params[:staging_project_copy_name], User.session!.id)
+        StagingProjectCopyJob.perform_now(@staging_workflow.project.name, params[:staging_project_project_name], params[:staging_project_copy_name], User.session!.id)
```

Comment:
That makes little sense, why do you change this?

Reply:
yes I was thinking about it too. Actually I wanted to go back to `perform_later`, but I've got sidetracked by another issue. `perform_later` is less less intrusive. The logging has low priority and shouldn't block other jobs to execute..